d8/d8batch/config/install/field.storage.node.field_feed_source_url.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 6d09794e-a019-42ae-9a43-67df9009c074
+langcode: en
```

Comment:
Core also contains language spec. settings in field storage conf. files.
http://cgit.drupalcode.org/drupal/tree/core/profiles/standard/config/install/field.storage.node.field_tags.yml?id=4f9265c